apps/dashboard/src/lib/components/Course/components/NewsFeed/NewFeedModal.svelte

```diff
@@ -110,11 +110,7 @@
     />
     <div class="flex items-center justify-end py-2">
       <div class="flex gap-2">
-        <PrimaryButton
-          label="Cancel"
-          variant={VARIANTS.OUTLINED}
-          onClick={() => ($isNewFeedModal.open = false)}
-        />
+        <PrimaryButton label="Cancel" variant={VARIANTS.OUTLINED} onClick={resetEditor} />
         <PrimaryButton label="Post" onClick={onPost} />
```

Comment:
@Chifez One last this is that, you need to track the `isLoading` on this post button and pass it as a prop, so it can be disabled when I click post.

Reply:
It's all done now